BSN.Resa.Mci.CallCenter.AgentApp.Data/CallbackQueue.cs

```diff
@@ -24,7 +24,7 @@ public CallbackQueue(IDatabaseFactory databaseFactory)
 
 		public CallbackDto Pop()
 		{
-			SortedSetEntry? callbackHolder = database.SortedSetPop(QUEUE_NAME);
+			SortedSetEntry? callbackHolder = database?.SortedSetPop(QUEUE_NAME);
```

Comment:
Why?

In construction of database, we have to check database to create normally and raise exception if needed, in all funcitons we can assume database is not null

Reply:
Please make workaround for this

Reply:
Please care it ;)

@Reza-Noei

Reply:
resolved.